modules/console-operator.adoc

```diff
@@ -1,16 +1,44 @@
 // Module included in the following assemblies:
 //
 // *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
 
 [id="console-operator_{context}"]
 = Console Operator
```

Comment:
```suggestion
ifdef::operator-ref[= Console Operator]
ifdef::cluster-caps[= Web console capability]
```
similar to Michael's PR, the title will show up slightly differently in the Operator reference page vs. the Cluster capabilities page(s).